### ISSUE TITLE
Floating colored label in menu doens't have color

### ISSUE THREAD

**Issue by etshy** [fomantic/Fomantic-UI#429]
# Bug Report

## Description
A label in a menu always have a grey color even with a color class. 

It seems to be fixed in the fomantic version provided in the fork though.

The `.ui.menu .item>.label` rule in my app and the `.ui.menu .item > .label` rule in jsfiddle are not the same.

In the jsFiddle : 
```
.ui.menu .item > .label {
    margin-left: 1em;
    padding: 0.3em 0.78571429em;
}
```

and in 2.7.1 on my app
```
.ui.menu .item>.label {
    background: #999;
    color: #fff;
    margin-left: 1em;
    padding: .3em .78571429em;
}
```

If this is indeed fixed, when will it be available for for update ?

## Testcase

[fiddle](https://jsfiddle.net/jkp8wnsa/1/) (working with the fomantic provided in the fiddle

## Screenshot (when possible)
 - My computed CSS
![My computed CSS](https://i.imgur.com/7kBAPH7.png)
 - My computed HTML
![My computed HTML](https://i.imgur.com/hPx7Gmg.png)
 - The result on my app
![The result on my app](https://i.imgur.com/cllr0UN.png)

## Version
2.7.1

EDIT : In the same time if we could have some otpions to change the label position it could be great.
Like some `bottom floating label` or `left top floating label` etc. with default position (when no position specified) like now, in top right.

**Comment by lubber-de** [comment]
I am confused : the fomantic version of the fiddle, where it is working, is the latest 2.7.1., so if you have different code, I wonder wh, you still say it's also 2.7.1. 😕
Could you prepare a fiddle where it is not working?
From what sources have you got or compiled the version you are using in your app?

Regarding additional options : They are available in the upcoming version 2.7.2 (`bottom` is indeed missing.. Maybe we add it too 😉 
https://github.com/fomantic/Fomantic-UI/pull/418

**Comment by lubber-de** [comment]
There also is a fix for floating labels in 2.7.2 regarding colors by https://github.com/fomantic/Fomantic-UI/pull/331

So you may try to compile the develop branch to test if this fixes your app also

**Comment by lubber-de** [comment]
Is your app using single component files instead of the whole semantic.css? I guess the menu.css is an older version (2.6.x) but label.css is 2.7.x then. The only Idea I have which would explain the behavior

**Comment by etshy** [comment]
a `npm info fomantic-ui` command returns this

```
**D:\PhpStormProjects\bookuto>npm info fomantic-ui

fomantic-ui@2.7.1 | MIT | deps: 35 | versions: 15**
...
```
so I guess I have the 2.7.1 installed.

I have this line in my `packages.json` : `"fomantic-ui": "^2.7.1"` too.

Here the fiddle with my copy of fomantic
https://jsfiddle.net/yd8fk2go/
I had to copy/paste the css file content in the css section, because I can't link to non-https ...


How can I get the develop branch in my app, to test ?
Is npm have the same options than composer to specify a branch ?

I tried to include the semantic.css from the jsFiddle in my app (and remove mine) and it worked, the color was good.
As I use webpack, i had another css files coimpiled in one so my page was messed up, but the color was here.

so I really think the semantic.css in the jsFiddle is not the same as the 2.7.1 provide by npm.

It confirms (more or less) that this will be fixed later in the 2.7.2. 
If you can confirm that, I guess this issue can be closed.
--- 
That's great for the new options !

**Comment by etshy** [comment]
Arf, I resintalled from scratch and a lot of my pages are totally messed.

I'll have to work about this, but the problem for this issue is fixed !

Must came from when I updated to 2.7.x, I guess.
